language/go/generate.go

```diff
@@ -371,9 +359,9 @@ type generator struct {
 	shouldSetVisibility bool
 }
 
-func newGenerator(c *config.Config, f *rule.File, rel string) *generator {
-	shouldSetVisibility := f == nil || !hasDefaultVisibility(f)
-	return &generator{c: c, rel: rel, shouldSetVisibility: shouldSetVisibility}
+func newGenerator(c *config.Config, args language.GenerateArgs) *generator {
```

Comment:
Nit: Let's actually delete `newGenerator` and just construct it with a literal.

I was thinking `newGenerator` and `language.GenerateArgs` should not be coupled, but if we pass in everything as parameters, it's a trivial constructor.